setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 110
+ignore = E133, E226, E228, E251, N802, N803, W391
```

Comment:
Is it an expected consequence of RFC-162 that we'll be including files like this in all packages? The RFC say so explicitly. It would be better if we could put it in a single, canonical location so that future changes can be simple and atomic. Maybe that's not possible...?